VSTS/Extension/vss-extension.json

```diff
@@ -1,74 +1,131 @@
+                                                                     {
+                                                                         "name":  "SonarQube Projects",
+                                                                         "endpointUrl":  "{{endpoint.url}}api/projects/index",
+                                                                         "resultSelector":  "jsonpath:$[*]"
+                                                                     }
+                                                                 ],
+                                                 "authenticationSchemes":  [
+                                                                               {
+                                                                                   "type":  "ms.vss-endpoint.endpoint-auth-scheme-basic",
+                                                                                   "inputDescriptors":  [
+                                                                                                            {
+                                                                                                                "id":  "username",
+                                                                                                                "name":  "Token",
+                                                                                                                "description":  "Authentication Token generated by SonarQube.",
+                                                                                                                "inputMode":  "textbox",
+                                                                                                                "validation":  {
+                                                                                                                                   "isRequired":  true,
+                                                                                                                                   "dataType":  "string"
+                                                                                                                               }
+                                                                                                            },
+                                                                                                            {
+                                                                                                                "id":  "password",
+                                                                                                                "name":  "Password",
+                                                                                                                "description":  "Password",
+                                                                                                                "inputMode":  "passwordbox",

```

Comment:
Good call we will make a  test with DEBUG=true and confirm everything is properly hidden.